wiki/Help_centre/Upgrading_to_lazer/en.md

```diff
@@ -11,6 +11,8 @@ The eventual goal is that this version will be released as an update which will
 
 "lazer" is a codename and will eventually be dropped as it becomes the primary release of the game. The rest of this document will refer to osu!(lazer) as "lazer" and osu!(stable) as "stable" for simplicity.
 
+*Notice: We are still figuring out game balance and mechanics. For now, **scores set on lazer should not be considered permanent**.*
```

Comment:
notices are supposed to be right below the heading, not at the end of a section

Reply:
What's the rational for this? Are they going to be displayed differently in the future?

Reply:
this is by convention, similar to [hatnotes](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#hatnote), mimicking how they're displayed on wikipedia:

![image](https://user-images.githubusercontent.com/36758269/210992746-2b75d4da-6ce6-4737-aa6a-a1b45e8f5f6d.png)

they play a similar role, and are sometimes stacked together. here's an example of a hatnote and a notice together:

![image](https://user-images.githubusercontent.com/36758269/210991838-613857e2-a151-41a5-8030-6c8ff5e175b2.png)

Reply:
they're not the same thing as these kinds of boxes if you were thinking of that

![image](https://user-images.githubusercontent.com/36758269/210994000-68cb4dd2-1c7d-46b8-8ae6-de82832a1811.png)